scroll-animations-1/Overview.bs

```diff
@@ -967,15 +987,14 @@ instead of document's scrolling Element for `auto`.
 
 'orientation' descriptor determines the scroll timeline's {{orientation}}.
 
-
 <pre class='descdef'>
-  Name: start
+  Name: scrollOffsets
```

Comment:
How about `scroll-offsets` to match Javascript definition of ScrollTimeline?

Reply:
Seems somewhat unnecessary to have the `scroll-` prefix when we're already inside `@scroll-timline`. IIRC we had `scroll-time-range` etc before, but the prefix was dropped.

_But_, if there are reasons for keeping the `scroll-` prefix on `scroll-offsets` specifically, then `scroll-offsets` WFM.

Reply:
There was a concern that offset has a different meaning in animation context, to specify the position of each keyframe. We may in fact want to add offsets as well as scrolloffsets in the future so that you could map scroll offsets to arbitrary animation offsets rather than having them uniformly distributed.

Reply:
Changed to scroll-offsets.

Reply:
I see, makes sense.